presto-orc/src/test/java/com/facebook/presto/orc/TestOrcWriterOptions.java

```diff
@@ -82,6 +93,9 @@ public void testToString()
         StreamLayout streamLayout = new StreamLayout.ByColumnSize();
         boolean integerDictionaryEncodingEnabled = false;
         boolean stringDictionarySortingEnabled = true;
+        boolean dwrfStripeCacheEnabled = false;
+        DwrfStripeCacheMode dwrfStripeCacheMode = FOOTER;
```

Comment:
nit: set this to INDEX and FOOTER, which is our most common type.

Reply:
Done